Formula/woff2.rb

```diff
@@ -0,0 +1,29 @@
+class Woff2 < Formula
+  desc "Utilities to create and convert Web Open Font File (WOFF) files"
+  homepage "https://github.com/google/woff2"
+
+  url "https://github.com/google/woff2/archive/v1.0.2.tar.gz"
+  sha256 "add272bb09e6384a4833ffca4896350fdb16e0ca22df68c0384773c67a175594"
+
+  bottle do
+    sha256 "01a88969d60bbb128a4a73a6b9b6d4c75f4ff08b7c3df45139fc73b7fc76d694" => :high_sierra
+  end
+
+  depends_on "cmake" => :build
+  depends_on "brotli"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    bin.install "woff2_info", "woff2_decompress", "woff2_compress"
```

Comment:
What does `make install` do if the three binaries are not installed?

Reply:
Surprisingly, no. It's both a library and a set of tools, and the tools don't get installed by default.

Reply:
Can you report it upstream? So that `make install` does what it should in the first place? (that will not block this PR, but would be nice)

Reply:
@rspeed please post the issue url here once you've reported it. Thanks!

Reply:
Done: google/woff2#113